agent_api_rest/src/identity/well_known/did_configuration.rs

```diff
@@ -0,0 +1,24 @@
+use agent_identity::{
+    service::{aggregate::ServiceResource, views::ServiceView},
+    state::{IdentityState, DOMAIN_LINKAGE_SERVICE_ID},
+};
+use agent_shared::handlers::query_handler;
+use axum::{
+    extract::State,
+    response::{IntoResponse, Response},
+    Json,
+};
+use hyper::StatusCode;
+
+#[axum_macros::debug_handler]
+pub(crate) async fn did_configuration(State(state): State<IdentityState>) -> Response {
+    // Get the DomainLinkageConfiguration if it exists.
```

Comment:
`DomainLinkageConfiguration` is just the Rust type (from identity.rs) of the object that is being returned. 

but I will change this comment to `// Get the DID Configuration Resource if it exists.` so that it beter reflects the
did-configuration spec terms :+1: